examples/passport-login/README.md

```diff
@@ -72,7 +76,7 @@ Open browser to http://localhost:3000
 1. Click on `Login` from the header menu, You will see various buttons under
    `Other login options`.
    - [Facebook](#Try-it-out-with-FaceBook)
-   - [google](#Try-it-out-with-Google)
+   - [Google](#Try-it-out-with-Google)
    - `Twitter` - not yet implemented
```

Comment:
same here. if we don't have an example yet, maybe we don't need it for now. similarly for other occurrences.

Reply:
well in the code we have given some url links which are not implemented, so that users can catch on later. It might confuse them, if we dont specify in the docs.